packages/app-desktop/services/autoUpdater/AutoUpdaterService.ts

```diff
@@ -0,0 +1,120 @@
+import { app, BrowserWindow } from 'electron';
+import { autoUpdater, UpdateInfo } from 'electron-updater';
+import log from 'electron-log';
```

Comment:
If possible please use our Logger class, so that log statements are sent to the expected file. Have a look how it's done in other files, but the pattern basically is `const logger = Logger.create('AutoUpdaterService')`